app/controllers/your-psp/get.controller.test.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const sinon = require('sinon')
+const getController = require('./get.controller')
+const gatewayAccountFixtures = require('../../../test/fixtures/gateway-account.fixtures')
+const { expect } = require('chai')
+const credentialId = 'a-valid-credential-id' 
+
+describe('Your psp GET controller', () => {
+  let req
+  let res
+  let next
+  const account = gatewayAccountFixtures.validGatewayAccount({
+    gateway_account_credentials: [
+      { 
+        state: 'ACTIVE', 
+        payment_provider: 'stripe', 
+        id: 100, 
+        external_id: credentialId 
+      }
+    ]
+  })
+  
+  beforeEach(() => {
+    req = {
+      correlationId: 'correlation-id',
+      account: account,
+      params: { credentialId },
+      flash: sinon.spy(),
+      url: '/kyc/'
+    }
+    res = {
+      redirect: sinon.spy(),
+      render: sinon.spy()
+    }
+    next = sinon.spy()
+  })
+
+
+  it('should pass the enableStripeOnboardingTaskList to the page ', async () => {
+
+    await getController(req, res, next)
+    
+    const pageData = res.render.args[0][1]
+    expect(pageData.enableStripeOnboardingTaskList).to.equal('true')
+    
+    sinon.assert.calledWith(res.render, 'your-psp/index')
+  })
+
+  it('should not pass the enableStripeOnboardingTaskList to the page when it is set to false ', async () => {
+    
+    await getController(req, res, next)
+    
+    const pageData = res.render.args[0][1]
+    pageData.enableStripeOnboardingTaskList = 'false' 
```

Comment:
You are actually setting it here on the page data and then testing it.
You have to remove this line.